.github/workflows/cd.yml

```diff
@@ -296,6 +298,44 @@ jobs:
         run: |
           npx lerna publish from-package --yes
 
+      - name: pack server visx
+        if: ${{ contains(steps.version-change.outputs.CHANGE, '@microsoft/teamsfx-server') }}
+        uses: nick-invision/retry@v2
+        with:
+          timeout_minutes: 10
+          max_attempts: 10
+          retry_on: error
+          command: |
+            sleep 20
+            cd ./packages/server
+            npx vsce package
```

Comment:
This package is not a VSIX package, so this may be "npm run package".

Reply:
thanks!